src/components/Dapp/index.js

```diff
@@ -5,12 +5,12 @@ import './index.css';
 
 export default class Dapp extends Component {
 
-    track = (e) => {
+    trackEventOpenListedDapp = (e) => {
         e.preventDefault();
-        trackEvent(ANALYTICS_EVENT_OPTS.CLICKS_DAPP, { 
-            'Dapp' : this.props.data.name,
-            'Url': this.props.data.url,
-            'Position': this.props.position + 1
+        trackEvent(ANALYTICS_EVENT_OPTS.OPEN_LISTED_DAPP, { 
+            'dapp_name' : this.props.data.name,
+            'dapp_url': this.props.data.url,
+            'position': this.props.position + 1
```

Comment:
The `position` was originally in the event. I wasn't sure if someone was using it for their analytics and since it doesn't affect our tracking plan I decided to just leave it